api/src/main/resources/workbench.yaml

```diff
@@ -872,6 +872,18 @@ paths:
           schema:
             $ref: '#/definitions/EmptyResponse'
 
+  /v1/workspaces/user-recent-resources:
+    get:
+      tags:
+        - userMetrics
+      description: Returns the resources accessed by user order by access time desc
+      operationId: getUserMetrics
```

Comment:
If the intent is for this to have more than just recentResources, I would call this function something different than `getUserMetrics`. Maybe `getRecentResources`?